packages/panzer/dof-mgr/test/cartesian_topology/tCartesianDOFMgr_DynRankView.cpp

```diff
@@ -170,154 +167,6 @@ TEUCHOS_UNIT_TEST(tCartesianDOFMgr_DynRankView, threed)
   out << "My Offset   = " << myOffset.x << " " << myOffset.y << " " << myOffset.z << std::endl;
   out << "My myElements = " << myElements.x << " " << myElements.y << " " << myElements.z << std::endl;
 
-  // check sharing locally on this processor
```

Comment:
@maartenarnst why are all the shared boundary tests being removed?

Reply:
It’s because they duplicate what is already in ‘ tCartesianDOFMgr’.